extension/event-page.html

```diff
@@ -0,0 +1,25 @@
+<!--
+Copyright (c) 2017 Google Inc. All rights reserved.
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt
+Code distributed by Google as part of this project is also
+subject to an additional IP rights grant found at
+http://polymer.github.io/PATENTS.txt
+-->
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Arcs: Chrome Plugin</title>
+
+  <script src="https://www.gstatic.com/firebasejs/4.2.0/firebase-app.js"></script>
```

Comment:
Does metadata-storage not load these resources?

Reply:
I'm not quite sure, to be honest. However, metadata-storage is now metadata-storage.js, so this is definitely needed now.